app/common/src/view/VerticalSwipeResponder.js

```diff
@@ -70,6 +79,7 @@ class VerticalSwipeResponder extends React.Component<Props> {
       //stop JS beautify collapse
       onStartShouldSetPanResponder: shouldSetResponder,
       onMoveShouldSetPanResponder: shouldSetResponder,
+      onPanResponderMove: this.handlePanResponderMove.bind(this),
```

Comment:
Hm 🙈, I am leaving the old-way binding here only because of consistency in this file but it would be nice to fulfil our todos:

> FIXME: **refactor** by throwing away all unnecessary code

🙂